orc8r/cloud/go/services/state/indexer/reindex/reindexer_singleton.go

```diff
@@ -60,6 +60,7 @@ func (r *reindexerSingleton) Run(ctx context.Context) {
 		}
 
 		clock.Sleep(reindexLoopInterval)
+		glog.Infof("Sleeping for %d seconds before looking for new jobs to reindex", reindexLoopInterval / time.Second)
```

Comment:
Nit: can just use the same time denomination as the var

Reply:
It's only 1 minute tho, don't know if 60 seconds would be more descriptive. I'm down to change tho!

Reply:
Is fine either way, my preference is simplicity of the codebase (it's lightly jarring to see the division -- makes me think "this is weird why are we dividing here" -- whereas just passing the var directly wouldn't set off any questions during a skim)

Reply:
Oh well in that case, I think this is better? Because there are 2 other options to not have division
1. Say that we sleeping for `60000000000` nanoseconds 
    - Unideal since that's hard to read
2. Hard code that we sleeping for a minute
    - Unideal since it's prone to human error --> someone might change the time, but forget to change the hard coding

Reply:
Either is fine. I prefer minutes since it's consistent with the same unit, but up to you

```
glog.Infof("...%v...", t.Seconds())
// or
glog.Infof("...%v...", t.Minutes())
```

Reply:
sure, i dont really mind either way